modules/BTBidAdapter.js

```diff
@@ -65,6 +66,22 @@ function request(buildRequest, imps, bidderRequest, context) {
   return request;
 }
 
+/**
+ * Processes a bid response using the provided build function, bid, and context.
+ *
+ * @param {Function} buildBidResponse - The function to build the bid response.
+ * @param {Object} bid - The bid object to include in the bid response.
+ * @param {Object} context - The context object containing additional information.
+ * @returns {Object} - The processed bid response.
+ */
+function bidResponse(buildBidResponse, bid, context) {
+  const bidResponse = buildBidResponse(bid, context);
+  const { seat } = context.seatbid || {};
+  bidResponse.btBidderCode = seat;
```

Comment:
not a big deal, but we probably could rename `btBidderCode` to something like `bidderSeat` or just `seat`.